app/models/Videos.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+from sqlalchemy.orm import relationship 
+from app import db 
+
+def default_value(video):
+    return video.get_current_parameters()["total_inventory"]
+
+class Video(db.Model):
+    __tablename__ = "videos"
+    video_id = db.Column(db.Integer, primary_key=True)
+    video_title = db.Column(db.String(120))
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer) 
+    available_inventory = db.Column(db.Integer, default=default_value)
```

Comment:
Like with `Customer` you can compute `available_inventory` with a relationship:

```suggestion
    rentals = db.relationship('Rental', back_populates='video', lazy=True)
```